client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -96,17 +96,19 @@ public final class ApiEndpointsV1
     public static final String KEYSPACE_TOKEN_MAPPING_ROUTE = API_V1 + PER_KEYSPACE + "/token-range-replicas";
 
     // Blob Transport Extension
-    public static final String JOB_ID_PATH_PARAM = ":jobId";
+    // todo: refactor other path param use cases; and replaceAll in requestURI building
```

Comment:
I think this can just be a JIRA, instead of a comment in the code.

Reply:
fair